nxpprog.py

```diff
@@ -1138,7 +1153,8 @@ def verify_image(self, flash_addr_base, image):
             end = end_addr if end_of_sector > end_addr else end_of_sector
             length = 4 * ((end - start) // 4)
 
-            log("Verify sector %i: Reading %d bytes from 0x%x" % (sector, length, start))
+            log("Verify sector %i: Reading %d bytes from 0x%x" % (sector, length, start),
```

Comment:
This is actually useful information for the user. This should be a normal log.

Reply:
Done.